test/matrix.lean

```diff
@@ -1,7 +1,3 @@
-/-
-manually ported from
-https://github.com/leanprover-community/mathlib/blob/4f4a1c875d0baa92ab5d92f3fb1bb258ad9f3e5b/test/matrix.lean
--/
```

Comment:
This looks like its left from a previous PR that I already reviewed; can you revert it here like you did there?